src/geometry/ArborX_Geometry.hpp

```diff
@@ -0,0 +1,37 @@
+/****************************************************************************
+ * Copyright (c) 2017-2025 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+#ifndef ARBORX_GEOMETRY_HPP
```

Comment:
I'd prefer an empty line before the header guards
```suggestion

#ifndef ARBORX_GEOMETRY_HPP
```